src/examples/forms/inputs/Validation.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Input, Message } from '@zendeskgarden/react-forms';
+
+const StyledRow = styled(Row)`
+  @media (max-width: ${props => props.theme.breakpoints.md}) {
+    display: flex;
+    flex-direction: column;
+
+    > div:not(:first-child) {
```

Comment:
Per previous comment, this should not be needed. But, in cases where it is, always prefer to reference a component when available, i.e. `${Col}:not(:first-child)`.